streaming/src/main/scala/org/apache/gearpump/streaming/appmaster/TaskManager.scala

```diff
@@ -121,7 +121,7 @@ private[appmaster] class TaskManager(
 
     case Event(RegisterTask(taskId, executorId, host), state@ TaskRegistrationState(register)) =>
       val client = sender
-      getMinClock.map(client ! StartClock(_))
+      getLatestStartClock.map(client ! StartClock(_))
```

Comment:
How can we make the fault-tolerance level configurable?
at-least once
exactly-once
at mostly once